src/main/java/jenkins/security/plugins/ldap/LDAPConfiguration.java

```diff
@@ -407,7 +430,9 @@ public String getDisplayName() {
         public FormValidation doCheckServer(@QueryParameter String value, @QueryParameter String managerDN, @QueryParameter Secret managerPasswordSecret,@QueryParameter String rootDN) {
             String server = value;
             String managerPassword = Secret.toString(managerPasswordSecret);
-
+            if(FIPS140.useCompliantAlgorithms() && !validateServerUrlIsSecure(server)){
```

Comment:
this check should be after line 436 (`if(!jenkins.get.hasPermissions`). Unless the user is an admin, the validation should not happen

Reply:
Done